src/wallet.rs

```diff
@@ -280,37 +289,42 @@ impl Wallet {
     self.settings.integration_test()
   }
 
-  pub(crate) fn is_mature(&self, rune: Rune, commit: &Transaction) -> Result<bool> {
-    let transaction = self
-      .bitcoin_client()
-      .get_transaction(&commit.txid(), Some(true))
-      .into_option()?;
+  fn is_after_minimum_height(&self, rune: Rune) -> bool {
+    rune
+      >= Rune::minimum_at_height(
+        self.chain().network(),
+        Height(u32::try_from(self.bitcoin_client().get_block_count().unwrap() + 1).unwrap()),
```

Comment:
This shouldn't unrwap:

```suggestion
        Height(u32::try_from(self.bitcoin_client().get_block_count()? + 1).unwrap()),
```

Reply:
done.